### PR TITLE
Minor aesthetical improvements

### PR DESCRIPTION
**Pull request by cpmachado**
Hey there,

Your plugin is simple, elegant, and suits my case. It's been interesting to check it as I myself don't know lua.

I just tweaked a few things, I hope you'll find it true.

Best wishes,
cpmachado

### CODE REVIEW

File: lua/oas-preview/main-module.lua
Comment (rusagaib) [inline lua/oas-preview/main-module.lua:1]: very thanks to this part tho, you actually clean up my messy code 🍻

**Review by rusagaib — APPROVED** [review]
fixing styles of codes thanks @cpmachado 🍻

**Review by rusagaib — APPROVED** [review]

**Review by rusagaib — APPROVED** [review]
looks good

**Review by rusagaib — APPROVED** [review]

**Review by rusagaib — APPROVED** [review]
thanks for your contribution @cpmachado 🍻

**Comment by cpmachado** [comment]
You welcome mate, glad to help. I'll look into helping a bit, when I have more time. Thanks for sharing.

**Pull request merged**